protocol/avdl/keybase1/teams.avdl

```diff
@@ -592,7 +594,16 @@ protocol teams {
   // Includes admins of the specified team only if they are also admins of ancestor teams.
   array<TeamMemberDetails> teamImplicitAdmins(int sessionID, string teamName);
 
-  AnnotatedTeamList teamList(int sessionID, string userAssertion, boolean all, boolean includeImplicitTeams);
+  // List known team memberships for given userAssertion.
+  AnnotatedTeamList teamList(int sessionID, string userAssertion, boolean includeImplicitTeams);
+
+  // List all team mates from all teams for current user. 
+  AnnotatedTeamList teamListTeammates(int sessionID, boolean includeImplicitTeams);
+
+  // List verified known team memberships for given userAssertion.
+  // This function is slower than teamList because it has to load
+  // every team.
+  AnnotatedTeamList teamListVerified(int sessionID, string userAssertion, boolean includeImplicitTeams);
```

Comment:
if changing the existing method names is hard, at least mention in a comment in the avdl on any of these functions if they are unverified